### ISSUE TITLE
Strange issue when having too many empty lines

### ISSUE THREAD

**Issue by vfonic** [browserify-rails/browserify-rails#127]
I've been banging my head against the wall, trying to setup browserify.

Here's the simple application.js file that didn't work:
```javascript
//= require jquery
//= require jquery_ujs
//= require vendor
//= require_directory ./public
//= require react
//= require react_ujs
//= require react_router
//= require react_router_ujs
//= require components

var greeter = require('./greeter');


```

Here's the one that worked:
```javascript
//= require jquery
//= require jquery_ujs
//= require vendor
//= require_directory ./public
//= require react
//= require react_ujs
//= require react_router
//= require react_router_ujs
//= require components

var greeter = require('./greeter');

```

The difference? The first one has two empty lines at the end of the file. :)

**Comment by vfonic** [comment]
After playing some more with the file, I think the issue might be due to files being only required via `require()`. When I require file using `//= require greeter`, then it works (or at least I couldn't repro the issue)

**Comment by cymen** [comment]
That is... odd. A tweak to the CommonJS detection got merged and released as 2.0.3. Do you still see the same problem with it?

**Comment by cymen** [comment]
Closing due to lack of response.
